src/components/ArticleForm/ArticleForm.js

```diff
@@ -68,7 +69,7 @@ const ArticleForm = ({
       <div className="text-area">
         <Editor
           className="editable"
-          data-placeholder="Tell your story here..."
+          data-placeholder={componentState.isCreated ? '' : 'Tell your story here...'}
```

Comment:
I modified this line of code because placeholder label overlays article data(body) to be modified. So on updating an article the placeholder gets hidden